### ISSUE TITLE
0.4.9 Method not found crash

### ISSUE THREAD

**Issue by michaeldimoudis** [Azure/DotNetty#414]
Hi just want to make you aware the recent 0.4.9 breaks MQTT on Microsoft.Azure.Devices.Client, 0.4.8 works fine.

```
System.MissingMethodException: Method not found: 'System.Threading.Tasks.Task DotNetty.Common.Concurrency.IEventExecutor.ScheduleAsync(System.Action`1<System.Object>, System.Object, System.TimeSpan)'.
   at Microsoft.Azure.Devices.Client.InternalClient.<>c.<ApplyTimeout>b__62_2(Task t)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
--- End of stack trace from previous location where exception was thrown ---
```

**Comment by michaeldimoudis** [comment]
Yep sure can. Please find minimal repro attached, .NET Core 2.1 console app, although crash also happens in 2.0.
[DotNettyRepro.zip](https://github.com/Azure/DotNetty/files/2293780/DotNettyRepro.zip)

**Comment by michaeldimoudis** [comment]
Hi I probably jumped the gun, I saw a new `Microsoft.Azure.Devices.Client` was just released and includes `DotNetty` 0.5.0. This combination is all working. So I don't think you need to look in to it further. Cheers.
